components/homepage/components/Creative/index.js

```diff
@@ -0,0 +1,94 @@
+import { Swiper, SwiperSlide } from "swiper/react";
+import SwiperCore, { Pagination } from "swiper";
+import "swiper/css";
+import "swiper/css/pagination";
+import {
+  StCreativeContainer,
+  StCreativeTitle,
+  StCreativeContant,
+  StCreativeContantLinks,
+  StCreativeContantInfo,
+  StCreativeContantSingleLink,
+  StCreativeFadeOne,
+  StCreativeFadeTwo,
+  StCreativeContantInfoBtn,
+  StCreativeContantImage,
+  StCreativeContantImageSwiper,
+} from "./styles";
+
+const LIST_ITEMS = [
+  "the objective of the campaign",
+  "hom much to spend",
+  "who to target",
+  "the type of advertising you'll see",
+];
+
+const index = () => {
```

Comment:
Change this component name from index to `Creative`, every component even if the file name is `index` has to be named as the main folder